src/lib/filter-when-format.js

```diff
@@ -0,0 +1,44 @@
+const docblock = require( 'jest-docblock' );
+
+function fileHasFormatPragma( file ) {
+	return file.source && 'format' in docblock.parse( file.source );
+}
+
+// If the file has the @format pragma at the beginning, it means it was formatted with
```

Comment:
I removed it -- the processor is documented elsewhere in more-or-less the same words.